src/components/shared/HomeSlider.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
```

Comment:
Three comments on this file:

- Why, and where are you importing material icons from. I thought we had settled on using react icons?
- I don't know why you chose this approach, perhaps you can let us know, but wouldn't it have been way more efficient, and time-effective to you use a powerful library for the slides?
- I just took a look on the UI, but I had a suggestions. It is really cool, but think about this. What if we wanted to pass clickable links to an image you are promoting? And also, should a slider remain visible in mobile screens? I am not sure, but I thought those kind of things would immediately become hidden in mobile.

@mucyoalexaxel, what do you think?

Reply:
I am using react icons, the reason for that import of material-icons; have been making an assessment of it with a comparison to react icons and then I forget to remove it.

And thank you for your review.